sources/metadata/everest/0.10.0-rc1.yaml

```diff
@@ -1 +0,0 @@
-version: 0.10.0-rc1
```

Comment:
Why do we need to delete the old RC?

Reply:
As discussed offline we don't know exactly how this would be handle and it's safer to just delete it for now.